spec/core_functions/color/alpha.hrx

```diff
@@ -96,6 +96,62 @@ a {
   b: opacity(10%);
 }
 
+<===>
+================================================================================
+<===> opacity/with_css_var/options.yml
+:ignore_for:
+  - libsass
+
+<===> opacity/with_css_var/input.scss
+a {b: opacity(var(--c))}
+
+<===> opacity/with_css_var/output.css
+a {
+  b: opacity(var(--c));
+}
+
+<===>
+================================================================================
+<===> opacity/with_calc/options.yml
+:ignore_for:
+  - libsass
+
+<===> opacity/with_calc/input.scss
+a {b: opacity(calc(1 + 2))}
+
+<===> opacity/with_calc/output.css
+a {
+  b: opacity(3);
+}
+
+<===>
+================================================================================
+<===> opacity/with_unquoted_calc/options.yml
+:ignore_for:
+  - libsass
+
+<===> opacity/with_unquoted_calc/input.scss
+a {b: opacity(unquote('calc(1)'))}
+
+<===> opacity/with_unquoted_calc/output.css
+a {
+  b: opacity(calc(1));
+}
+
+<===>
+================================================================================
+<===> opacity/with_clamp/options.yml
+:ignore_for:
+  - libsass
+
+<===> opacity/with_clamp/input.scss
+a {b: opacity(clamp(1rem, 2.5vw, 2rem))}
+
+<===> opacity/with_clamp/output.css
+a {
+  b: opacity(clamp(1rem, 2.5vw, 2rem));
+}
+
```

Comment:
We probably don't need to test every different type of calculation, especially since we'll probably be adding a bunch of new ones shortly.